mappings/net/minecraft/world/gen/decorator/PredicatedDecorator.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_6661 net/minecraft/world/gen/decorator/PredicatedDecorator
```

Comment:
hmm, we usually call it "conditional"... "predicated" sounds like you pass in a predicate object while you actually override the abstract method